blocks.md

```diff
@@ -161,33 +176,23 @@ Key | Values | Required | Default
 `device` | The device in `/sys/class/power_supply/` to read from. When using UPower, this can also be `"DisplayDevice"`. | No | `"BAT0"`
 `driver` | One of `"sysfs"` or `"upower"`. | No | `"sysfs"`
 `interval` | Update interval, in seconds. Only relevant for `driver = "sysfs"`. | No | `10`
-`format` | A string to customise the output of this block. See below for available placeholders. Text may need to be escaped, refer to [Escaping Text](#escaping-text). | No | `"{percentage}%"`
-`full_format` | Same as `format` but for when the battery is full. | No | `"{percentage}%"`
-`missing_format` | Same as `format` but for when the specified battery is missing. | No | `"{percentage}%"`
+`format` | A string to customise the output of this block. See below for available placeholders. Text may need to be escaped, refer to [Escaping Text](#escaping-text). | No | `"{percentage}"`
+`full_format` | Same as `format` but for when the battery is full. | No | `"{percentage}"`
+`missing_format` | Same as `format` but for when the specified battery is missing. | No | `"{percentage}"`
 `allow_missing` | Don't display errors when the battery cannot be found. Only works with the `sysfs` driver. | No | `false`
 `hide_missing` | Completely hide this block if the battery cannot be found. Only works in combination with `allow_missing`. | No | `false`
 `info` | Minimum battery level, where state is set to info. | No | `60`
 `good` | Minimum battery level, where state is set to good. | No | `60`
 `warning` | Minimum battery level, where state is set to warning. | No | `30`
 `critical` | Minimum battery level, where state is set to critical. | No | `15`
 
-#### Deprecated Options
-
-Key | Values | Required | Default
-----|--------|----------|--------
-`show` | Deprecated in favour of `format`. Show remaining `"time"`, `"percentage"` or `"both"`. | No | `"percentage"`
-`upower` | Deprecated in favour of `device`. When `true`, use the Upower D-Bus driver. | No | `false`
-
-The `show` option is deprecated, and will be removed in future versions. In the meantime, it will override the `format` option when present.
-
 #### Available Format Keys
 
-Placeholder | Description
-------------|-------------
-`{percentage}` | Battery level, in percent
-`{bar}` | The current battery level in a bar chart
```

Comment:
Is it possible to keep {bar} around and deprecate it instead? Then we can remove it in a future release.

Reply:
I'm not sure if it's possible.

Reply:
OK then we can document the new way to do it somewhere.

Reply:
In case of battery block I've already added an example.

Reply:
Let's document these breaking changes in NEWS.md. I started it off with a couple.